docs/recipes/handlebars.md

```diff
@@ -58,32 +97,39 @@ gulp.task('serve', ['styles', 'templates', 'fonts'], () => {
 Edit your `serve` task so that (a) editing an `.hbs` file triggers the `templates` task, and (b) the browser is reloaded whenever a `.js` file is generated in `.tmp/templates`:
 
 ```diff
- gulp.task('serve', ['styles', 'templates', 'fonts'], () => {
+gulp.task('serve', ['styles', 'templates', 'partials', 'scripts', 'fonts'], () => {
    ...
    gulp.watch([
-     'app/*.html',
-     '.tmp/styles/**/*.css',
-+    '.tmp/templates/**/*.js',
-     'app/scripts/**/*.js',
-     'app/images/**/*'
-   ]).on('change', reload);
-
-   gulp.watch('app/styles/**/*.scss', ['styles', reload]);
-+  gulp.watch('app/templates/**/*.hbs', ['templates', reload]);
-   gulp.watch('bower.json', ['wiredep', 'fonts', reload]);
+    'app/*.html',
+    'app/images/**/*',
+    '.tmp/fonts/**/*',
++   '.tmp/templates/**/*.js',
+  ]).on('change', reload);
+
+  gulp.watch('app/styles/**/*.scss', ['styles']);
++  gulp.watch('app/templates/*.hbs', ['templates']);
++  gulp.watch('app/templates/partials/*.hbs', ['partials']);
+  gulp.watch('app/scripts/**/*.js', ['scripts']);
+  gulp.watch('app/fonts/**/*', ['fonts']);
+  gulp.watch('bower.json', ['wiredep', 'fonts']);
  });
 ```
 
 ## Usage
 
-Put `.hbs` files in `app/templates`, and include them in your HTML as if they're `.js` files.
-
-Example – if you've written a template at `app/templates/foo.hbs`:
+Put your `.hbs` files in `app/templates` and `app/templates/partials` (if you using partials), and include in your HTML after the bower components:
 
 ```html
-<script src="bower_components/handlebars/handlebars.runtime.js"></script>
-<script src="templates/foo.js"></script>
-<script src="scripts/main.js"></script>
+  <!-- build:js scripts/vendor.js -->
+  <!-- bower:js -->
+  ...
+  <!-- endbower -->
+  <!-- endbuild -->
+
+  <!-- build:js templates/index.js -->
```

Comment:
Maybe I'm missing something here, but why should we open another comment block, why not just merge templates and partials with the `scripts/main.js` block next to it?